platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,204 @@
+                featuresService.uninstallFeature("admin-modules-installer",
+                        EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+            }
+        } catch (Exception e) {
+            printError("An error occurred while trying to perform post-install operations");
+            LOGGER.debug("Error while trying to run the post-install start and stop operations.", e);
+        }
+    }
+
+    private void installFeatures(FeaturesService featuresService, List<String> installFeatures) {
+        if (installFeatures != null) {
+            printSectionHeading("Installing Features");
+            installFeatures.forEach(feature -> {
+                printItemStatusPending("Installing: ", feature);
+                boolean installSuccess = executeAsSystem(() -> {
+                    try {
+                        featuresService.installFeature(feature,
+                                EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+                    } catch (ResolutionException e) {
+                        printItemStatusFailure("Install Failed: ", feature);
+                        LOGGER.error(String.format("An error occurred installing feature: %s", feature), e);
+                        return false;
+                    }
+                    return true;
+                });
+                if (installSuccess) {
+                    printItemStatusSuccess("Installed: ", feature);
+                }
+            });
+        }
+    }
+
+    private void uninstallFeatures(FeaturesService featuresService, List<String> uninstallFeatures) {
+        if (uninstallFeatures != null) {
+            printSectionHeading("Uninstalling Features");
+            uninstallFeatures.forEach(feature -> {
+                printItemStatusPending("Uninstalling: ", feature);
+                executeAsSystem(() -> {
+                    Feature featureObject = featuresService.getFeature(feature);
+                    featuresService.uninstallFeature(featureObject.getName(),
+                            featureObject.getVersion(),
+                            EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+                    return true;
+                });
+                try {
+                    if (!featuresService.isInstalled(featuresService.getFeature(feature))) {
+                        printItemStatusSuccess("Uninstalled: ", feature);
+                    } else {
+                        printItemStatusFailure("Uninstall Failed: ", feature);
+                    }
+                } catch (Exception e) {
+                    printError("An error occurred when checking if feature uninstalled successfully");
+                    LOGGER.error("Error occurred when checking if feature was uninstalled successfully: ", e);
+                }
+            });
+        }
+    }
+
+    private void stopBundles(BundleService bundleService, List<String> stopBundleNames) {
```

Comment:
Let's add a comment indicating stopping bundle is temporary workaround and support for it will be removed in the future.